2048/2048/src/main/java/com/appcoins/eskills2048/MainGame.java

```diff
@@ -105,6 +101,17 @@ public void newGame() {
         mView.invalidate();
     }
 
+    private void startPeriodicOpponentUpdate() {
```

Comment:
this method is running twice on the first time the user starts a game:
1. from onCreate method
2. from onResume method

Reply:
yeap, you're right. i'm always forgetting about activity life-cycle. thank you